.circleci/config.yml

```diff
@@ -14,46 +14,88 @@ orbs:
 
 jobs:
 
-  build:
+  build_and_test:
     <<: *defaults
+
     steps:
+      - run: &setup_env_vars
+          name: Setup Environment Variables
+          command: |
+            export PROJECT_NAME=ml-recipe-midi-generator
+            export PROJECT_PATH_STORAGE=storage:build-$PROJECT_NAME-$CIRCLE_WORKFLOW_ID
+            export IMAGE_NAME=neuromation/$PROJECT_NAME
+            export IMAGE_REF=$IMAGE_NAME:build-$CIRCLE_WORKFLOW_ID
+
+            echo "PROJECT_NAME=$PROJECT_NAME" >> $BASH_ENV
+            echo "PROJECT_PATH_STORAGE=$PROJECT_PATH_STORAGE" >> $BASH_ENV
+            echo "IMAGE_NAME=$IMAGE_NAME" >> $BASH_ENV
+            echo "IMAGE_REF=$IMAGE_REF" >> $BASH_ENV
+
       - setup_remote_docker
       - checkout
-      - run:
-          name: Install Dependencies
+      - run: &install_dependencies
+          name: Install Dependencies, Configure Environment
           command: |
+            pip install --user --upgrade pip neuromation
```

Comment:
should this be in `.circleci/requirements.txt` and upgraded below?

Reply:
sure thanks